packages/calcite-components/src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -368,6 +371,8 @@ export class InputTimePicker
    */
   @Event({ cancelable: true }) calciteInputTimePickerChange: EventEmitter<void>;
 
+  @Event() calciteInputTimePickerLangUpdated: EventEmitter<void>;
```

Comment:
@eriklharper should this be an internal event since its not really for public use?

If we do want to make this for public use maybe we can have a more generic event like `calciteLangChange` and it could specify the component in the event details?

Reply:
Thanks for the feedback, yes, this should probably start out as an internal event, and to your thought about making this for public use, I also thought of making the event even more generically-named that we could probably build into an interface that any component that needs to asynchronously load any kind of extra resources can implement, perhaps extending the `LoadableComponent` interface to include loading any extra assets after initial mount.  Thoughts?